### DISCUSSION TITLE
Double-click profile name to load the profile, instead of clicking load button

### DISCUSSION THREAD

**Discussion by Invincibear** [Rem0o/FanControl.Releases#1948] [Ideas]
Title sums it up, it would be nice to be able to just double-click, especially since I have to switch profiles daily.

**Comment by Rem0o** [comment]
Do you mean double clicking an icon say on your desktop? You can!

Create a shortcut for FanControl.exe, and add to the "target" in properties:

```
FanControl.exe -c theConfigYouWant.json
```

**Comment by Invincibear** [comment]
I mean in the UI, when I click the menu -> load profile -> select the
profile name, I'd like to double-click the profile name instead of
selecting it and clicking the load button.

**Reply by Rem0o** [reply]
Ah, well sure, why not. I'll add it.   

You can also right click the tray icon and select your config from there. No need to keep the UI open, it takes more memory also having it open.

**Comment by Invincibear** [comment]
Thanks for the taskbar tip, and the feature add!
